ros_installation.md

```diff
@@ -1,46 +1,97 @@
-# Install Gazebo and ROS
+# Installing Gazebo with ROS
 
-This document provides an overview about the options to use different versions
-of ROS in combination with different versions of Gazebo. It is recommended to read
+This document provides guidance on using different versions
+of ROS in combination with different versions of Gazebo. We recommended to read
 it before installing [ros_gz](https://github.com/gazebosim/ros_gz).
 
-# Important! Simple analysis for a quick and correct decision
+# Picking the "Correct" Versions of ROS & Gazebo
+
+If this is your first time using ROS and Gazebo, and you are not following
+specific instructions or a tutorials, we recommend using the latest long term
+support (LTS) versions of both ROS and Gazebo for the best user
+experience. Successive versions of ROS and Gazebo releases are named
+alphabetically, and the documentation for each version of ROS and Gazebo will
+indicate if it is an LTS version. It is worth noting that each version of ROS
+works best with a specific version of each Tier 1 platform. Tier one platforms
+are platforms / host operating systems that are used in the development of ROS
+and Gazebo.All of this information is outlined in
+[REP-2000](https://www.ros.org/reps/rep-2000.html). To summarize, the best user
+experience is to use the latest LTS version of ROS and the Tier 1 platform /
+operating system recommend for that version of ROS. If you host operating system
```

Comment:
Nitpick: this is a long paragraph that could be a bit hard to start the document with. Can we split it in a different one here:
```
operating system recommend for that version of ROS.

If you host operating system
```

Reply:
Fixed.